demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -36,6 +42,140 @@ def __init__(self, release_notes_file_path, modified_files=None, pack_name=None,
+            headers[content_type] = [item.replace('New:', '').strip() for item in content_items]
+
+    def validate_content_type_header(self, content_type: str) -> bool:
+        """
+            Validate that the release notes 1st headers (the content type) are a valid content entity.
+        Args:
+            content_type: (str) - The content type to validate.(e.g. Integrations, Playbooks, etc.)
+        Return:
+            True if the content type is valid, False otherwise.
+        """
+        # Get all the content type headers
+        rn_valid_headers = list(map(lambda rn_valid_header: rn_valid_header.lower(), RN_HEADER_BY_FILE_TYPE.values()))
+
+        if content_type.lower() not in rn_valid_headers:
+            error_message, error_code = Errors.release_notes_invalid_content_type_header(content_type=content_type, pack_name=self.pack_name)
+            if self.handle_error(error_message, error_code, self.release_notes_file_path):
+                return False
+        return True
+
+    def validate_content_item_header(self, content_type: str, headers: Dict) -> bool:
+        """
+            Validate the 2nd headers (the content items) are exists in the pack and having the right display name.
+        Args:
+            content_type: (str) - The content type to validate.(e.g. Integrations, Playbooks, etc.)
+            headers: (Dict) - The release notes headers to validate, the structure is content type -> headers.(e.g. Integrations -> [header1, header2])
+        Return:
+            True if the content item is valid, False otherwise.
+        """
+        is_valid = True
+        # Special rule for the Mapper, as the Mapper is under the classifier directory.
+        content_type_dir_name = content_type.replace(' ', '') if content_type != RN_HEADER_BY_FILE_TYPE[
+            FileType.MAPPER] else RN_HEADER_BY_FILE_TYPE[FileType.CLASSIFIER]
+        content_type_path = os.path.join(self.pack_path, content_type_dir_name)
+
+        content_type_dir_list = []
+        try:
+            for file in os.listdir(content_type_path):
+                if os.path.isdir(os.path.join(content_type_path, file)):
+                    file_path = os.path.join(content_type_path, file, f'{file}.yml')
+                else:
+                    file_path = os.path.join(content_type_path, file)
+                if file_path.endswith('.yml') | file_path.endswith('.json'):
+                    content_type_dir_list.append(file_path)
+        except FileNotFoundError:
+            error_message, error_code = Errors.release_notes_invalid_content_type_header(content_type=content_type,
+                                                                                         pack_name=self.pack_name)
+            if self.handle_error(error_message, error_code, self.release_notes_file_path):
+                is_valid = False
+
+        content_type_dir_list = list(map(lambda item: get_display_name(item), content_type_dir_list))
```

Comment:
it's ok - list comprehension is more readable IMO

Reply:
Agreed, but less efficient